samples/cellular/gnss/README.rst

```diff
@@ -86,7 +86,13 @@ GNSS satellite acquisition can also be assisted by providing factory almanac, GP
 Using this information, GNSS can calculate which satellites it should search for and what are the expected Doppler frequencies.
 
 The sample includes a factory almanac that is written to the file system when the sample starts.
-The date for the factory almanac generation is in the :file:`factory_almanac.h` file.
+There are two versions of the factory almanac:
+
+* :file:`factory_almanac_v2.h`: Used by nRF9160 devices.
+* :file:`factory_almanac_v3.h`: Used by nRF91x1 devices.
+
+The date when the factory almanac was generated can be found in the header files.
```

Comment:
```suggestion
The date for generating the factory almanac is in the header files.
```

Reply:
Blah, bad morning. My proposal sucks. Maybe your text is better. But still "is found" or just "is".